Exercise_6/ReversePolishNotation.js

```diff
@@ -0,0 +1,44 @@
+// const process = require('process');
+// console.log(calculate(process.argv[2]));
+
+function calculate(argument,extraOperators) {
+    const invalidArgumentError = new Error('The argument ' + argument + ' is invalid');
+    const signs = argument.split(' ');
+    const stack=[];
+
+    function isNumber(sign) {
+        return !isNaN(parseInt(sign)) && isFinite(sign);
+    }
+
+    function isValidOperator(sign) {
+        const isValidOperatorRegex = new RegExp(/^[+\-*]$/);
+        return isValidOperatorRegex.test(sign);
+    }
+
+
+    signs.reduce((stack,sign) => {
```

Comment:
Reduce should **return** value and not mutate existing one. Because right now your reduce function works like forEach loop.